packages/hash/api/src/model/knowledge/hashInstance.model.ts

```diff
@@ -169,12 +201,54 @@ export default class extends EntityModel {
   ): Promise<void> {
     const { userModel, actorId } = params;
 
-    const adminLink = await LinkModel.get(graphApi, {
-      sourceEntityId: this.entityId,
-      linkTypeId: SYSTEM_TYPES.linkType.admin.schema.$id,
-      targetEntityId: userModel.entityId,
-    });
+    const outgoingAdminLinkEntityModels = await LinkEntityModel.getByQuery(
+      graphApi,
+      {
+        all: [
+          {
+            equal: [
+              { path: ["leftEntity", "uuid"] },
```

Comment:
Same as above about `ownedById`

Reply:
Addressed in https://github.com/hashintel/hash/pull/1435/commits/8aa3152488cf511718d875ba0dfa8cbfcdf30d2d